src/http/DataMetadataEndpoints.ts

```diff
@@ -1,4 +1,9 @@
 import express, { Request, Response } from 'express'
+import { Storage } from '../storage/Storage'
+
+const parseIntIfExists = (x: string) => {
```

Comment:
should the type of x be then `string | undefined`?

Reply:
Fixed